source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -72,6 +72,23 @@ If you want to override GOV.UK Frontend's styles with your own styles, `@import`
 
 If your project uses GOV.UK Frontend toolkit, GOV.UK Template or GOV.UK Elements, you can [configure GOV.UK Frontend to work with them](/compatibility-mode/).
 
+### Silence deprecation warnings in Dart Sass
```

Comment:
Is 'silence' the most direct way of saying what it is users can do here? What about 'turn off'?

Also, does this title suggest users can silence all deprecation warnings (whereas line 90 says it's only warnings caused by dependencies)?

Reply:
> Is 'silence' the most direct way of saying what it is users can do here? What about 'turn off'?

It matches the terminology used in the Sass docs here:

> This is useful for silencing deprecation warnings that you can’t fix on your own.
>
> – https://sass-lang.com/documentation/js-api#quietdeps

I think it's a verb that users would tend to associate with warnings / errors / logs, so I think it helps to front-load the heading so users who are looking for it would recognise it quickly. It also seems slightly more accurate than turn off – the warnings are still 'happening', it's just you've opted to ignore them.

> Also, does this title suggest users can silence all deprecation warnings (whereas line 90 says it's only warnings caused by dependencies)?

Will update to 'Silence deprecation warnings from dependencies in Dart Sass'?